src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -28,6 +28,7 @@
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core" />
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="$(MicrosoftVisualStudioImageCatalogVersion)" />
```

Comment:
Was this something already being consumed due to a dependency, or is it something actually new?

Reply:
It was pulled in as a transitive dependency but no longer there.